### PR TITLE
Fix tags project

### PR DESCRIPTION
**Pull request by RachelBaram**
Changing tags for all past blogs

### CODE REVIEW

**Comment by amitaibu** [comment]
Can be deployed?

**Comment by RachelBaram** [comment]
Not yet. I have more fixes to add.

**Comment by RachelBaram** [comment]
@amitaibu Now it can be deployed. Thanks!

File: app/_posts/2015-10-01-elmlang.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ tags:
   - Headless Drupal
   - Elm
   - "Drupal-planet"
+  - Shop Talk
```
Comment (amitaibu) [inline app/_posts/2015-10-01-elmlang.md]: What does it mean?

Comment (RachelBaram) [inline app/_posts/2015-10-01-elmlang.md]: Speaking with the guys, we determined this is the tag we will use for technical content.  
"shop talk - talk about your business that only others in the same business can understand"

Comment (amitaibu) [inline app/_posts/2015-10-01-elmlang.md]: Isn't it 90% of our posts?

Comment (RachelBaram) [inline app/_posts/2015-10-01-elmlang.md]: Yes, for now. We would like to lower this % by creating a lot more non-tech content. Once we get the website to display the content nicely based on tags #440, the role of this tag will be more important.  Techies who would like to browse Gizra blogs but cut out the non-tech stuff can do it easily without having to choose a specific technology tag.

Comment (amitaibu) [inline app/_posts/2015-10-01-elmlang.md]: I think it should be removed. IMO, when we reach the point we have lower % we can introduce it. I don't re-call seeing it anywhere. Also, a dev learning about Drupal, not necessarily cares about our Elm stuff.

**Comment by RachelBaram** [comment]
I don't feel strongly about this so no problem.

File: app/_posts/2015-12-30-hedley-overview.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ tags:
 permalink: "/content/elm-hedley-overview"
 layout: post
 published: true
---- 
+---
```
Comment (RachelBaram) [inline app/_posts/2015-12-30-hedley-overview.md]: not sure what I did here and not sure how to fix it.

**Comment by RachelBaram** [comment]
@amitaibu Done with taking out Shop Talk.

**Comment by RachelBaram** [comment]
@amitaibu Can you deploy please. ;)

**Comment by amitaibu** [comment]
Deployed, thanks.

**Pull request merged**